bag/datasets/bag/batch_gob.py

```diff
@@ -18,6 +17,496 @@
 log = logging.getLogger(__name__)
 
 
+class CodeOmschrijvingDataTask(batch.BasicTask):
```

Comment:
Ik weet niet of ik data in code nu de beste oplossing vind, is het niet beter deze referentietabellen bij de data op te slaan (en ze daar te laten beheren?)

Reply:
In GOB willen ze juist niets meer van de codes weten. Ze worden hier toegevoegd voor backwards compatibiliteit (zodat de FE blijft werken) 

Later kunnen ze eventueel worden verwijderd indien er geen andere klanten zijn die daarvan afhankelijk zijn